scenarios/kubernetes_build.py

```diff
@@ -31,6 +32,36 @@ def check(*cmd):
     subprocess.check_call(cmd)
 
 
+def get_gcs_path(args):
+    # xref https://github.com/kubernetes/release/blob/master/push-build.sh
+    # RELEASE_BUCKET=${FLAGS_bucket:-"kubernetes-release-dev"}
+    bucket = args.release if args.release else "kubernetes-release-dev"
```

Comment:
No need for python's horrible ternary pattern:
`bucket = args.release or 'kubernetes-release-dev'`